Intermitent failures of test_set_uint

**Describe the bug**
```
 ___________________ TestIndexing.test_set_uint[float64-100] ____________________

self = <indexing_test.TestIndexing object at 0x7fbdc6054400>, prob_size = 100
dtype = <class 'numpy.float64'>

    @pytest.mark.parametrize("prob_size", pytest.prob_size)
    @pytest.mark.parametrize("dtype", NUM_TYPES)
    def test_set_uint(self, prob_size, dtype):
        test_size = prob_size // 2
        ikeys, ukeys = key_arrays(prob_size)
        pda = value_array(dtype, prob_size)
    
        # set [int] = val with uint key and value
        pda[np.uint(2)] = np.uint(5)
        assert pda[np.uint(2)] == pda[2]
    
        # set [slice] = scalar/pdarray
        pda[:test_size] = -2
        assert pda[ukeys].to_list() == pda[ikeys].to_list()
        pda[:test_size] = ak.cast(ak.arange(test_size), dtype)
        assert pda[ukeys].to_list() == pda[ikeys].to_list()
    
        # set [int] = val with uint key and value
        val = value_scalar(dtype, prob_size)[0]
        pda[np.uint(2)] = val
>       assert pda[2] == val
E       assert -0.02417804175662752 == -0.024178041756627522

tests/indexing_test.py:79: AssertionError
```

**Additional context**
This error message was pulled from the CI.   arkouda multi-dim 2.2

--- a/tests/indexing_test.py
+++ b/tests/indexing_test.py
@@ -76,7 +76,7 @@ class TestIndexing:
         # set [int] = val with uint key and value
         val = value_scalar(dtype, prob_size)[0]
         pda[np.uint(2)] = val
-        assert pda[2] == val
+        assert np.allclose(pda[2], val)
 
         # set [slice] = scalar/pdarray
         pda[:prob_size] = val

